validator-cli/src/utils/arbToEthState.ts

```
@@ -0,0 +1,208 @@
+import { SequencerInbox__factory } from "@arbitrum/sdk/dist/lib/abi/factories/SequencerInbox__factory";
+
+// https://github.com/prysmaticlabs/prysm/blob/493905ee9e33a64293b66823e69704f012b39627/config/params/mainnet_config.go#L103
+const slotsPerEpochEth = 32;
+const secondsPerSlotEth = 12;
+
+const getBlocksAndCheckFinality = async (
+  EthProvider: JsonRpcProvider,
+  ArbProvider: JsonRpcProvider,
+  veaEpoch: number,
+  veaEpochPeriod: number
+): Promise<[Block, Block, Boolean] | undefined> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)